From 6aa06ccb0e409948d5b4eaf3a8d80b9abdd8dd4c Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Thu, 22 Oct 2015 10:31:08 +0800
Subject: [PATCH 558/602] MA-6450-6 Added compile macro for BCM4339 wifi

Added BOARD_SUPPORT_BCM_WIFI macro for wpa_supplicant
compile.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 evk_6sl/BoardConfig.mk     | 2 ++
 sabresd_6dq/BoardConfig.mk | 2 ++
 sabresd_6sx/BoardConfig.mk | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/evk_6sl/BoardConfig.mk b/evk_6sl/BoardConfig.mk
index ec43ced..b29b0c0 100644
--- a/evk_6sl/BoardConfig.mk
+++ b/evk_6sl/BoardConfig.mk
@@ -50,6 +50,8 @@ BOARD_WPA_SUPPLICANT_PRIVATE_LIB_RTL        := lib_driver_cmd_rtl
 BOARD_HOSTAPD_PRIVATE_LIB_BCM               := lib_driver_cmd_bcmdhd
 BOARD_WPA_SUPPLICANT_PRIVATE_LIB_BCM        := lib_driver_cmd_bcmdhd
 
+BOARD_SUPPORT_BCM_WIFI  := true
+
 #for intel vendor
 ifeq ($(BOARD_WLAN_VENDOR),INTEL)
 BOARD_HOSTAPD_PRIVATE_LIB                := private_lib_driver_cmd
diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index b0d4936..7fe33d0 100644
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -62,6 +62,8 @@ BOARD_HOSTAPD_PRIVATE_LIB_RTL               := lib_driver_cmd_rtl
 BOARD_WPA_SUPPLICANT_PRIVATE_LIB_RTL        := lib_driver_cmd_rtl
 BOARD_HOSTAPD_PRIVATE_LIB_BCM               := lib_driver_cmd_bcmdhd
 BOARD_WPA_SUPPLICANT_PRIVATE_LIB_BCM        := lib_driver_cmd_bcmdhd
+
+BOARD_SUPPORT_BCM_WIFI  := true
 #for intel vendor
 ifeq ($(BOARD_WLAN_VENDOR),INTEL)
 BOARD_HOSTAPD_PRIVATE_LIB                := private_lib_driver_cmd
diff --git a/sabresd_6sx/BoardConfig.mk b/sabresd_6sx/BoardConfig.mk
index 7dff636..e8f3504 100644
--- a/sabresd_6sx/BoardConfig.mk
+++ b/sabresd_6sx/BoardConfig.mk
@@ -51,6 +51,8 @@ BOARD_WPA_SUPPLICANT_PRIVATE_LIB_RTL        := lib_driver_cmd_rtl
 BOARD_HOSTAPD_PRIVATE_LIB_BCM               := lib_driver_cmd_bcmdhd
 BOARD_WPA_SUPPLICANT_PRIVATE_LIB_BCM        := lib_driver_cmd_bcmdhd
 
+BOARD_SUPPORT_BCM_WIFI  := true
+
 #for intel vendor
 ifeq ($(BOARD_WLAN_VENDOR),INTEL)
 BOARD_HOSTAPD_PRIVATE_LIB                := private_lib_driver_cmd
-- 
1.8.0

